src/databricks/labs/remorph/install.py

```diff
@@ -322,12 +487,44 @@ def run(
         return config
 
     @classmethod
-    def install_rct(cls):
-        RCTInstaller.install()
+    def install_bladerunner(cls):
+        local_name = "bladerunner"
+        pypi_name = "databricks-bb-plugin"
+        TranspilerInstaller.install_from_pypi(local_name, pypi_name)
 
     @classmethod
     def install_morpheus(cls):
-        MorpheusInstaller.install()
+        java_version = cls.get_java_version()
+        if java_version < 110:
+            logger.warning(
```

Comment:
shouldn't this be error?

Reply:
It's a warning because because it doesn't prevent installation of `bladerunner`

Reply:
I keep forgetting we install both together.